<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormFirst</class>
 <widget class="QWidget" name="MainFormFirst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>681</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1061</width>
    <height>681</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainFormFirst {
	border-image: url(:/qrc/res/main_bk.png)  50 00 0 0 repeat;
}
QLineEdit {
font: bold 36px; 
background-color: rgba(255, 255, 255,160);
border-radius: 0px;
border: 0px solid rgba(39, 22, 22, 254);
}
QGroupBox
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
#widget_back
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
#widget_back_3
{
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}

#groupBox1 QLabel {
color: rgb(255, 255, 255);
font: bold 18px;
background-color: transparent;
}
QLineEdit {font: bold 24px;}
#labelSumTitle {
color: rgb(255, 255, 255);
font: bold 24px;
padding-left:2px;
}
#labelClassTitle {
color: rgb(255, 255, 255);
font: bold 24px;
padding-left:2px;
}

#MainHomePage QTableWidget {
	background-color: transparent;
	font: bold 24px;	
	color: rgb(234, 234, 116);
}
#MainHomePage QHeaderView{  
	background-color: transparent;
}
#MainHomePage QHeaderView::section {	
	background-color: transparent;
	color: rgb(255, 255, 255);
	font: bold 16px;
	border: 0px solid #6c6c6c;
	height: 40px;
}
#MainHomePage QTableWidget::item {
	background-color: rgba(130, 157, 255, 20);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>751</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>35</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>工序代码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_MO_DispatchNo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_MO_ProcName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_MO_ProcCode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>派工单号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>工序名称</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_MainOrderFlag">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>主 单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MO_TotalNum">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btItemUp">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>10</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold  12px;
	color: rgb(255, 255, 127);
border-top-left-radius:20px;
border-top-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);</string>
   </property>
   <property name="text">
    <string>上单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btItemNext">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>110</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold  12px;
	color: rgb(255, 255, 127);
border-bottom-left-radius:20px;
border-bottom-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);</string>
   </property>
   <property name="text">
    <string>下单</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>961</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(255, 255, 255);
font: bold 18px;
background-color: transparent;
}
QLineEdit {
font: bold 24px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>产品编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_ItemNo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>工单号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_MO">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_MouldNo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模具编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>派工数量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_DispatchQty">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_SocketNum1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>该件模穴数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_PCS_BadData">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>次品总数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>生产总数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Count">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(255, 255, 255);
font: bold 18px;
background-color: transparent;
}
QLineEdit {
font: bold 24px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>员工编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>职务</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btItemNextItem">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>420</y>
     <width>81</width>
     <height>191</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold  12px;
	color: rgb(255, 255, 127);
border-bottom-left-radius:20px;
border-bottom-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);</string>
   </property>
   <property name="text">
    <string>下一件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btItemUpItem">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>81</width>
     <height>191</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold  12px;
	color: rgb(255, 255, 127);
border-top-left-radius:20px;
border-top-right-radius:20px;
border: 1px solid rgba(255, 255, 255,100);
background-color: rgba(85, 170, 127,100);</string>
   </property>
   <property name="text">
    <string>上一件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
